.github/ISSUE_TEMPLATE/3-feature-request.yml

```diff
@@ -6,26 +6,25 @@ labels:
   - discovery needed
 
 body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: >-
+        A clear and concise description of what the new feature should do
+      placeholder: >-
+        `<rh-jazz-hands>` should be let users choose the era of Jazz the element represents
+    validations:
+      required: true
 
-- type: textarea
-  id: description
-  attributes:
-    label: Description
-    description: >-
-      A clear and concise description of what the new feature should do
-    placeholder: >-
-      `<rh-jazz-hands>` should be let users choose the era of Jazz the element represents
-  validations:
-    required: true
-
-- type: textarea
-  id: solution
-  attributes:
-    label: Suggested solution
-    description: >-
-      Describe the solution you'd like: a clear and concise description of what you want to happen.
-  validations:
-    required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Suggested solution
+      description: >-
+        Describe the solution you'd like: a clear and concise description of what you want to happen.
+    validations:
+      required: true
 
   - type: textarea
```

Comment:
Looks like screenshot textarea is tabbed in 1 too many

Reply:
I don't see this indentation issue in VS Code:

<img width="598" alt="image" src="https://github.com/RedHat-UX/red-hat-design-system/assets/1890950/e4686e61-ac2a-4c85-8a06-3244f4b550e9">